README.md

```diff
@@ -47,11 +50,47 @@ function YourComponent() {
       <CioAutocomplete 
         apiKey="key_M57QS8SMPdLdLx4x"
         onSubmit={(e) => {console.log(e)}}
-      />
     </div>
   );
 ```
 
+## Shopify
+
+### ⚡ Installation & Quick Start
+
+The Constructor autocomplete component is available as part of the [Constructor.io Shopify App](https://apps.shopify.com/constructor-connect)
+
+After installing the app, you can use the Constructor autocomplete component by clicking 'Add Section' in your theme editor and adding the Constructor autocomplete liquid component
+
+![installation gif](https://files.readme.io/80b6698f86ab7c4695450dad50b2d30a205116d61ef1468d2af6ea4281a16cf0-autocomplete.gif)
```

Comment:
This is not rendering for me 🤔 
[Link](https://github.com/Constructor-io/constructorio-ui-autocomplete/tree/ci-4257-os-ui-autocomplete-add-github-readme-integration-modes)
<img width="840" alt="Screenshot 2025-05-05 at 11 37 20 AM" src="https://github.com/user-attachments/assets/43aa11f7-fd08-472a-9966-5fd7926e3017" />

Reply:
🤔 I can't get this GIF to work. It works on local. I wonder if theres some github limitation.

Reply:
Yeah it might be refusing to pull a gif from a different source. What do you think about downloading the gif and putting it [here](https://github.com/Constructor-io/constructorio-ui-autocomplete/tree/main/assets)?

Reply:
🤔 Yea I'll do that. I was trying some other third party gifs, they were working fine not sure what's special about this one.